pyproject.toml

```diff
@@ -0,0 +1,56 @@
+[tool.poetry]
+name = "lda"
+version = "2.0.0"
+description = "Topic modeling with latent Dirichlet allocation"
+authors = ["lda developers <lda-users@googlegroups.com>"]
+license = "MPL 2.0"
+readme = "README.rst"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
+    "Programming Language :: C",
+    "Programming Language :: Cython",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+]
+
+[tool.poetry.build]
+generate-setup-file = true
+script = "build.py"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<=3.12"
```

Comment:
To be able to support Python 3.12, we have to drop Python <3.8, because Poetry says:

```
The current project's Python requirement (>=3.8,<=3.12) is not compatible with some of the required packages Python requirement:
  - numpy requires Python <3.13,>=3.9, so it will not be satisfied for Python >=3.8,<3.9
```